values.yaml

```diff
@@ -401,13 +390,19 @@ container_images:
         registry: replicas.mirror.gpkg.io/proxy-docker-io
         repository: curlimages/curl
         tag: 8.16.0@sha256:463eaf6072688fe96ac64fa623fe73e1dbe25d8ad6c34404a669ad3ce1f104b6
+  app_gluekube_ccm:
+    gluekube_ccm:
+      image:
+        registry: replicas.mirror.gpkg.io/proxy-quay-io
```

Comment:
switch to new registries from alanis. These old ones need to get deleted as we roll out this platform helm chart version